content/300-guides/200-deployment/301-edge/550-deploy-to-deno-deploy.mdx

```diff
@@ -307,10 +309,9 @@ Use the GitHub repository to add your application to Deno Deploy.
 1. Select a GitHub organization or user and then select a repository.
 1. Select a production branch and select **Automatic** mode so that Deno Deploy can deploy every time you push a change to the repository.
 1. Select `index.ts` as the entry point to your project.
-1. To define the Data Proxy connection string, click **Add Env Variable**.
+1. To define the Accelerate connection string, click **Add Env Variable**.
    1. For **KEY**, enter `DATABASE_URL`.
-   1. For **VALUE**, paste the Data Proxy connection string.
-      ![Deno Deploy - project parameters](./images/550-02-deploy-to-deno-project-parameters.png)
```

Comment:
Why remove this?

Reply:
I specifically commented on the removed image line. We used to have an image here, now we don't.

Reply:
I removed it so that in case Deno's UI changes, we don't have to update it again. Hence, kept just the text for it. I 'll add that back in.